WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsHardwareDetailView.swift

```diff
@@ -20,19 +25,242 @@ struct PointOfSaleSettingsHardwareDetailView: View {
                     }
                 }
             }
-            .navigationDestination(for: PointOfSaleSettingsView.HardwareDestination.self) { destination in
-                VStack(spacing: POSSpacing.medium) {
-                    Image(systemName: destination.icon).font(.largeTitle)
+            .navigationDestination(for: NavigationDestination.self) { destination in
+                switch destination {
+                case .hardware(.cardReaders):
+                    cardReadersView
+                case .hardware(.scanners):
+                    scannersView
+                case .scanner:
```

Comment:
Could we remove `scanner` from the `NavigationDestination` then?

Reply:
Yes, I thought would be needed but it isn't since only buttons are there now. Updated: https://github.com/woocommerce/woocommerce-ios/pull/16038/commits/9fdcf3635a420db28939b49820634c86e1012e45